app/services/idv/steps/ssn_step.rb

```diff
@@ -26,10 +29,19 @@ def invalid_state?
         flow_session[:pii_from_doc].nil?
       end
 
+      def updating_ssn
```

Comment:
let's file a FIXME ticket and see if we can find a way to share code with these two SSN steps? In case we ever add a 3rd?